GA/utils.py

```
@@ -0,0 +1,82 @@
+        Must have the n_objectives and POPULATION_size attributes.
+
+    Returns
+    -------
+    - p : int
+        The number of points needed
+    """
+
+    try:
+        m = cfg.n_objectives
+    except AttributeError as e:
+        raise AttributeError(f"Configuration object is missing required attribute: {e}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)